docs/pt/docs/deployment.md

```diff
@@ -0,0 +1,394 @@
+# Implantação
+
+Implantar uma aplicação **FastAPI** é relativamente fácil.
+
+Existem vários modos de realizar o _deploy_ dependendo de seu caso de uso específico e as ferramentas que você utiliza.
+
+Você verá mais sobre alguns modos de fazer o _deploy_ nas próximas seções.
+
+## Versões do FastAPI
+
+**FastAPI** já está sendo utilizado em produção em muitas aplicações e sistemas. E a cobertura de teste é mantida a 100%. Mas seu desenvolvimento ainda está andando rapidamente.
```

Comment:
Eu mudaria de 
`Mas seu desenvolvimento ainda está andando rapidamente.`
para
`Mas seu desenvolvimento continua andando rapidamente.`

Reply:
Acho que encaixa melhor com a versão em inglês de `. But its development is still moving quickly.`

Reply:
Boa!